src/template.txt

```diff
@@ -158,7 +158,7 @@ define(<%= imports %>, function(<%= args %>){
 		<%= ases %>
 	};
 	var isNW = (function(){
-		try{var nr = System._nodeRequire; return nr && nr('nw.gui') !== 'undefined';}catch(e){return false;}
+		try{var nr = steal._nodeRequire; return nr && nr('nw.gui') !== 'undefined';}catch(e){return false;}
```

Comment:
@matthewp wasn't sure if this change was required for a Steal upgrade. Is this change potentially problematic?